data/themes/darktable.css

```diff
@@ -179,6 +179,12 @@
 @define-color timeline_text_bg_color alpha(black, .8);
 @define-color timeline_text_fg_color alpha(white, .9);
 
+@define-color range_graph_color alpha(@timeline_fg_color, 0.6);
+@define-color range_bg_color alpha(@bg_color, 0.2);
+@define-color range_cursor_color alpha(@fg_color, .75);
+@define-color range_sel_color alpha(@timeline_fg_color, 0.22);
```

Comment:
Finally, setting 0.5 here would be better. By using it more, it's strange to have selected items darker. Setting 0.5 instead of 0.22 make it lighter and make it easier to show selected items.

Reply:
Sorry, my bad, this also need to change color by bg one. So replace line 185 by:

`@define-color range_sel_color alpha(@timeline_bg_color, 0.5);`